src/commands/dev.ts

```diff
@@ -178,12 +184,13 @@ export async function command({cwd, config}: DevOptions) {
 
   function getUrlFromFile(fileLoc: string): string | null {
     for (const [dirDisk, dirUrl] of mountedDirectories) {
-      if (fileLoc.startsWith(dirDisk)) {
+      if (fileLoc.startsWith(dirDisk + path.sep)) {
```

Comment:
why this change? Wouldn't a correct file match return true in either scenario?

Reply:
To prevent file `/foo/barbaz.js` from matching dir `/foo/bar/`. I hope I didn't miss something.

Reply:
clever!